### PR TITLE
chore: update blocky-rtc toolbox xml to json

### LINKED ISSUE #1252: Update blockly-rtc to use JSON toolbox definitions instead of xml
**Category**
 - Examples

**Component**

blockly-rtc

**Is your feature request related to a problem? Please describe.**

Many of the examples and plugins use XML toolboxes instead of the new JSON API. We should have our example code use the new APIs to best model what other developers can do with Blockly.

**Describe the solution you'd like**

Update the code in blockly-samples to use the JSON toolbox API.

**Additional context**

To work on this issue, first read the [background information](https://developers.google.com/blockly/guides/contribute/samples) about how to contribute to samples and verify your changes.

1. Learn about [defining a toolbox](https://developers.google.com/blockly/guides/configure/web/toolbox) and read about the XML format vs the JSON format.
2. In blockly-samples, navigate to the blockly-rtc example's directory. Some examples can be started with `npm run start` and some can be opened directly by opening the `index.html` file in a browser. The README may give specific instructions, or you can look in the `package.json` to see what scripts are defined. Note you may need to run `npm install` in the example directory. Either way, open the example's demo page to observe the toolbox and its contents.
3. Write a new copy of the toolbox definition in the JSON format instead of the XML format. Where exactly it should be declared will depend on the example. It will most likely make sense to declare the toolbox definition close to the location where it is used, i.e. where `Blockly.inject` is called.
4. Refresh the demo page and make sure the toolbox looks the same as it did before. You can compare against the version in master on the [samples site](https://google.github.io/blockly-samples/) to make sure the behavior hasn't changed. Note that not all demos are published on the samples site so it may not be there.
5. Follow the rest of the instructions in the Contributing docs to verify your change and open a PR.

### PR DESCRIPTION
**Pull request by Samridhi-98**
**Description**

-  Updated `examples/blocky-rtc`
-  Updated blocky version in package json
-  Rewrite the toolbox for websocket and http

Fixes #1252

### CODE REVIEW

File: examples/blockly-rtc/src/http/index.js [hunk 1]
```diff
@@ -23,31 +23,100 @@
...
-        toolbox: document.getElementById('toolbox'),
-        media: 'media/',
-      });
+    {
+      toolbox: toolbar,
+      media: 'media/',
+    });
   const workspaceClient = new WorkspaceClient(
-      workspace.id, getSnapshot, getEvents, writeEvents);
+    workspace.id, getSnapshot, getEvents, writeEvents);
```
Comment (BeksOmega) [inline examples/blockly-rtc/src/http/index.js]: Please revert this whitespace change (and similar). All line continuation should be indented by 4 spaces =)

File: examples/blockly-rtc/src/http/index.js [hunk 2]
```diff
@@ -67,7 +136,7 @@ document.addEventListener('DOMContentLoaded', async () => {
    * @private
    */
   function runEvents_(eventQueue) {
-    eventQueue.forEach((workspaceAction)=> {
+    eventQueue.forEach((workspaceAction) => {
```
Comment (BeksOmega) [inline examples/blockly-rtc/src/http/index.js:139]: This whitespace change is great though!

File: examples/blockly-rtc/src/http/index.js [hunk 3]
```diff
@@ -23,31 +23,100 @@
...
 
 import * as Blockly from 'blockly';
-import {getSnapshot, getEvents, writeEvents} from './workspace_client_handlers';
-import {getPositionUpdates, sendPositionUpdate} from './user_data_handlers';
+import { getSnapshot, getEvents, writeEvents } from './workspace_client_handlers';
+import { getPositionUpdates, sendPositionUpdate } from './user_data_handlers';
 import UserDataManager from '../UserDataManager';
 import WorkspaceClient from '../WorkspaceClient';
 
+const toolbar = {
```
Comment (BeksOmega) [inline examples/blockly-rtc/src/http/index.js]: Could you rename this (and the other reference) to `toolbox` for consistency?

File: examples/blockly-rtc/src/http/index.js [hunk 4]
```diff
@@ -23,31 +23,100 @@
  */
 
 import * as Blockly from 'blockly';
-import {getSnapshot, getEvents, writeEvents} from './workspace_client_handlers';
-import {getPositionUpdates, sendPositionUpdate} from './user_data_handlers';
+import { getSnapshot, getEvents, writeEvents } from './workspace_client_handlers';
```
Comment (BeksOmega) [inline examples/blockly-rtc/src/http/index.js]: Please revert this whitespace change.

**Review by BeksOmega — CHANGES_REQUESTED** [review]
Looks great! Just a few nit picky whitespace changes haha. Sorry :/ the linter in samples isn't always perfect! Thank you for your work on this :D

File: examples/blockly-rtc/src/http/index.js (hunk 1 above)
Comment (BeksOmega) [reply examples/blockly-rtc/src/http/index.js]: Sorry! What I meant is that this line actually needs to be indented /more/ not less hehe. Whenever you continue a line of code to a new line it needs to be indented by 4 spaces =)

File: examples/blockly-rtc/src/http/index.js (hunk 4 above)
Comment (BeksOmega) [reply examples/blockly-rtc/src/http/index.js]: I think this still needs to be fixed as well! Please remove the spaces inside the braces so it looks like:

```
import {getSnapshot, getEvents, writeEvents}
```

File: examples/blockly-rtc/src/http/index.js (hunk 1 above)
Comment (Samridhi-98) [reply examples/blockly-rtc/src/http/index.js]: Done! 
Sorry for troubling 😅

**Review by BeksOmega — APPROVED** [review]
Sweetness! Ty for the fixes :D White space is hard lol. Really appreciate all of the work you've been doing!!

**Pull request merged**